src/Sensors/Baro/BMP280.cpp

```diff
@@ -3,40 +3,31 @@
 namespace mmfs
 {
 
-    BMP280::BMP280() : bmp()
+    BMP280::BMP280(const char *name) : bmp()
     {
-        setName("BMP280");
+        setName(name);
     }
 
-    bool BMP280::initialize()
+    bool BMP280::init()
     {
         if (!bmp.begin())
         { // hardware I2C mode, can pass in address & alt Wire
             // Serial.println("Could not find a valid BMP390 sensor, check wiring!");
             return initialized = false;
         }
 
-        delay(1000);
```

Comment:
Was there a reason this delay was in here before?

Reply:
Most likely to help the sensor stabilize, but by the time this function is run, the board will have been powered for multiple seconds.